integrations/apache/documentation.yaml

```diff
@@ -0,0 +1,91 @@
+exporter_type: sidecar
+app_name_short: Apache HTTP
+app_name: {{app_name_short}}
+app_site_name: Apache HTTP
+app_site_url: https://httpd.apache.org/
+exporter_name: the Apache HTTP exporter
+exporter_pkg_name: httpd_exporter
+exporter_repo_url: https://github.com/Lusitaniae/apache_exporter
+dashboard_available: true
+minimum_exporter_version: v0.11.0
+multiple_dashboards: false
+dashboard_display_name: {{app_name_short}} Prometheus Overview
+config_mods: |
+  apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: httpd
+  data:
+    httpd.conf: |
+      ...
+  +   <Location "/server-status">
+  +       SetHandler server-status
+  +   </Location>
+  +   LoadModule status_module modules/mod_status.so
+      ...
+  ---
+  apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    name: httpd
+  spec:
+    replicas: 1
+    selector:
+      matchLabels:
+  +     app.kubernetes.io/name: httpd
+    template:
+      metadata:
+        labels:
+  +       app.kubernetes.io/name: httpd
+      spec:
+        containers:
+        - name: httpd
+          image: httpd:2.4
+          ports:
+          - containerPort: 80
+            name: httpd
+  +       volumeMounts:
+  +       - mountPath: /usr/local/apache2/conf/httpd.conf
+  +         subPath: httpd.conf
+  +         name: httpd
+  +     - name: httpd-exporter
+  +       image: lusotycoon/apache-exporter:v0.11.0
+  +       ports:
+  +       - containerPort: 9117
+  +         name: prometheus
+  +       command: ["/bin/apache_exporter"]
+  +       args: ['--scrape_uri=http://localhost/server-status?auto', '--telemetry.address=:9117', '--telemetry.endpoint=/metrics']
+  +     volumes:
+  +     - name: httpd
+  +       configMap:
+  +         name: httpd
+  +         items:
+  +         - key: httpd.conf
+  +           path: httpd.conf 
+additional_install_info: |
+  These instructions assume you already have a working {{app_name_short}}
+  installation and want to modify it to include an exporter. Apache HTTP server
+  can be configured to serve metrics on `/server-status` by modifying the configuration
+  with a new Location directive and by loading the [status_module](https://httpd.apache.org/docs/2.4/mod/mod_status.html){:class=external}.
+podmonitoring_config: |
+  apiVersion: monitoring.googleapis.com/v1
+  kind: PodMonitoring
+  metadata:
+    name: httpd
+    labels:
+      app.kubernetes.io/name: httpd
+      app.kubernetes.io/part-of: google-cloud-managed-prometheus
+  spec:
+    endpoints:
+    - port: prometheus
+      scheme: http
+      interval: 30s
+      path: /metrics
+    selector:
+      matchLabels:
+        app.kubernetes.io/name: httpd
+additional_prereq_info: |
+  Ensure that the values of the `port` and `matchLabels` fields match those of the {{app_name_short}} pods you want to monitor.
+alerts_config: {% includecode content_path="stackdriver/docs/managed-prometheus/exporters/configs/httpd_alerts.yaml" %}
```

Comment:
IIUC, this field refers to a separate yaml file that contains the alerts. Did you mean to add that file as part of this PR or will that be a separate PR? If it is the latter, we can remove this for now.

Reply:
The case here was that the existing alerts were not easily ported over using the available prometheus metrics. I've removed both sections. Thank you for the feedback, I appreciate it!